.github/workflows/setup_environment.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Ensure the script is running as root
+if [ "$(id -u)" -ne 0 ]; then
+    echo "This script must be run as root"
+    exit 1
+fi
+
+# Python setup
+apt-get update -yqq
+apt-get install -yqq python3-pip
+pip3 install tox
+
+# Microk8s setup
+snap install microk8s --channel=1.28/stable --classic
```

Comment:
The microk8s channel gets hidden in this script - can we expose it in the workflow?  What about having:
```suggestion
snap install microk8s --channel=$MICROK8S_CHANNEL --classic
```

and then in the workflow above call with:
```yaml
    - name: Setup operator environment
      run: sudo bash .github/workflows/setup_environment.sh
      env:
        MICROK8S_CHANNEL=1.28/stable
```